README.md

```
@@ -148,3 +151,35 @@ $ fanyi config set LLM_MODEL_ID your-model-id         // set LLM_MODEL_ID
 If you want to use other LLMs, make sure you have properly set `LLM_API_KEY` `LLM_API_BASE_URL` and `LLM_MODEL_ID`. Also, the api endpoint should be [OpenAi Compatible](https://platform.openai.com/docs/api-reference/chat/create).
+
+## Agent Skill
+
+This repo ships an [Agent Skill](https://github.com/afc163/fanyi/tree/main/skills/fanyi) so AI agents (Claude Code, Claude.ai, and other tools that support the Agent Skills spec) can call `fanyi` for you automatically — just say "翻译一下 serendipity" or "和谐 用英语怎么说" and the agent runs `fanyi` under the hood, returning the full iciba / youdao / LLM results.
+
+### Install
+
+First make sure the CLI is installed globally so the skill can invoke it:
+
+```bash
+$ npm i fanyi -g
```

markdownlint-cli2 (0.22.1)

[warning] Dollar signs used before commands without showing output

(MD014, commands-show-output)